### PR TITLE
Add tests for cgroups validations

### PR DESCRIPTION
**Pull request by dmah42**

### CODE REVIEW

File: auraed/src/cells/cell_service/cells/cgroups/cpuset/cpus.rs [hunk 1]
```diff
@@ -76,12 +67,9 @@ mod tests {
     #[test]
     fn test_validation_success(input: &str) {
         let input: Cpus = Cpus::new(input.into());
```
Comment (future-highway) [inline auraed/src/cells/cell_service/cells/cgroups/cpuset/cpus.rs:69]: I know this is from before, but it seems weird that we use `Cpus::new` and then call `.into_inner()` on ln 71. Why don't we just call `input.to_string()` on 71?

We do this in other tests as well.

File: auraed/src/cells/cell_service/validation.rs [hunk 2]
```diff
@@ -294,3 +294,224 @@ impl From<ValidatedExecutable> for super::executables::ExecutableSpec {
...
+            "field",
+            Some("parent"),
+        );
+        assert!(validated.is_err());
+    }
+
+    #[test]
+    fn test_cell_type_cpu_weight_too_large() {
+        let validated = CellValidator::validate_cpu(
+            Some(CpuController { weight: Some(20000), max: None }),
```
Comment (future-highway) [inline auraed/src/cells/cell_service/validation.rs:338]: Is it better to test the first invalid value (i.e., 10001)?

File: auraed/src/cells/cell_service/executables/executable_name.rs [hunk 3]
```diff
@@ -6,6 +6,13 @@ use validation::{ValidatedField, ValidationError};
 #[derive(Debug, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
 pub struct ExecutableName(String);
 
+#[cfg(test)]
+impl ExecutableName {
+    pub(crate) fn new(name: String) -> Self {
```
Comment (future-highway) [inline auraed/src/cells/cell_service/executables/executable_name.rs:11]: I think this should have the `#[cfg(test)]` attribute.

**Review by future-highway — COMMENTED** [review]
Comment re `#[cfg(test)]` is the only one that stopped my approving.

**Comment by krisnova** [comment]
Can we please address @future-highway's requests and then we can merge this in?

File: auraed/src/cells/cell_service/executables/executable_name.rs (hunk 3 above)
Comment (dmah42) [reply auraed/src/cells/cell_service/executables/executable_name.rs:11]: it doesn't inherit from the impl?

File: auraed/src/cells/cell_service/validation.rs (hunk 2 above)
Comment (dmah42) [reply auraed/src/cells/cell_service/validation.rs:338]: a QA tester walks into a bar, orders 1 beer, 10001 beers, 20000 beers, NaN beers...

File: auraed/src/cells/cell_service/executables/executable_name.rs (hunk 3 above)
Comment (future-highway) [reply auraed/src/cells/cell_service/executables/executable_name.rs:12]: I think you are right. It should inherit; I missed that it was above. I do prefer it on the fn though (but wouldn't have not approved had I seen it).

**Review by future-highway — APPROVED** [review]

**Pull request merged**